doc/sphinxman/source/oeprop.rst

```diff
@@ -166,3 +168,18 @@ grid point, in the order that the grid was specified, while the *Exvals*,
 electric field, respectively; all of these arrays can be iterated and
 manipulated using standard Python syntax.  For a complete demonstration of this
 utility, see the :srcsample:`props4` test case.
+
+
+.. _`sec:oeprop_mbis`:
+
+
+Minimal Basis Iterative Stockholder
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Minimal Basis Iterative Stockholder (MBIS) partitioning is one of many procedures
+that produce atomic one-electron densities from the molecular one-electron density.
+Calling MBIS in Psi4 will calculate atomic charges, as well as dipoles, quadrupoles, and
+octupoles. The allowed number of iterations and convergence criteria for the stockholder 
+algorithm is controlled by |globals__mbis_maxiter| and |globals__mbis_convergence|. Note 
+that the density is partitioned on a molecular grid, so any settings related to the number
+or type of grid points applies. (Associated Paper: T. Verstraelen et al. "Minimal Basis Iterative Stockholder: Atoms in Molecules for Force-Field Development". *J. Chem. Theory and Comput.* doi: 10.1021/acs.jctc.6b00456.)
```

Comment:
Instead of writing out the paper information, you can directly link to the citation that you added to the bibliography. The syntax looks something like [Verstraelen:2016]_. You can look at other files in doc/sphinxman/source/ to get an idea